tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -208,23 +192,108 @@ def test_download_dependencies_skip_deps(
             "version_in_nexus": "6.5.5-external-gitcommit-78032157f5c1655436829017bbda787565b48c30",
         },
     ]
-    proxy_repo_url = npm.get_npm_proxy_repo_url(1)
+
+    request_id = 1
+    proxy_repo_url = npm.get_npm_proxy_repo_url(request_id)
     download_dir = tmpdir.join("deps")
     download_dir.mkdir()
+
     general_js.download_dependencies(
         Path(download_dir), deps, proxy_repo_url, {"@angular/animations@8.2.14"}
     )
 
-    mock_run_cmd.assert_called_once()
-    # This ensures that the skipped dependency is not downloaded
-    expected_npm_pack = [
-        "npm",
-        "pack",
-        "@angular-devkit/architect@0.803.26",
-        "rxjs@6.5.5-external-gitcommit-78032157f5c1655436829017bbda787565b48c30",
+    # dep_1
+    dep1_source_path = os.path.join(
+        download_dir,
+        "angular-devkit-architect-0.803.26.tgz",
+    )
+    dep1_dest_path = os.path.join(
+        download_dir,
+        "@angular-devkit/architect/angular-devkit-architect-0.803.26.tgz",
+    )
+    # dep_2
+    dep2_source_path = os.path.join(
+        download_dir,
+        "rxjs-6.5.5-external-gitcommit-78032157f5c1655436829017bbda787565b48c30.tgz",
+    )
+    commit_hash = "-gitcommit-78032157f5c1655436829017bbda787565b48c30"
+    dep2_dest_path = os.path.join(
+        download_dir,
+        f"github/ReactiveX/rxjs/rxjs-6.5.5-external{commit_hash}.tgz",
+    )
+
+    # This ensures that the skipped dependency is skipped
+    assert len(mock_move.call_args_list) == 2
+
+    mock_move.assert_has_calls(
+        [
+            mock.call(dep1_source_path, dep1_dest_path),
+            mock.call(dep2_source_path, dep2_dest_path),
+        ],
+    )
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "dep": "chai@4.2.0",
+            "p_url": "http://nexus:8081/repository/1/chai/-/chai-4.2.0.tgz",
+            "tarball_name": "chai-4.2.0.tgz",
+        },
+        {
+            "dep": "assertion-error@1.1.0",
```

Comment:
it looks like we don't test any "interesting" cases - anything where the `@` and `/` replacements would be relevant